### DISCUSSION TITLE
scheduling (repetitive) async tasks in FastAPI

### DISCUSSION THREAD

**Discussion by subhadipsjsc** [fastapi/fastapi#3660] [Questions]
what do you recommend for scheduling (repetitive) async tasks in FastAPI every 30 seconds? what is your recommendation?

**Comment by lesnik512** [comment]
Seems like [Repeated Tasks](https://fastapi-utils.davidmontague.xyz/user-guide/repeated-tasks/) from [fastapi-utils](https://github.com/dmontagu/fastapi-utils/)

**Reply by subhadipsjsc** [reply]
thanks
